.travis.yml

```diff
@@ -11,7 +11,9 @@ matrix:
   - python: 3.7
     dist: xenial
     sudo: true
-install: python setup.py install
+install: 
+- if [[ $TRAVIS_PYTHON_VERSION == 3.3 ]]; then pip install --upgrade setuptools==39.2.0; fi
```

Comment:
Any idea why this is now needed? The build used to run fine on Travis

Reply:
Because https://travis-ci.org/seatsio/seatsio-python/jobs/540966258
After searching the web and experimenting, i got that working. Not sure if it was a temporary issue tho